base/reflection.jl

```diff
@@ -738,7 +738,8 @@ function length(mt::MethodTable)
 end
 isempty(mt::MethodTable) = (mt.defs === nothing)
 
-uncompressed_ast(m::Method) = uncompressed_ast(m, isdefined(m, :source) ? m.source : m.generator.inferred)
+uncompressed_ast(m::Method) = isdefined(m,:source) ? uncompressed_ast(m, m.source) :
```

Comment:
Conditionally returning the source code from a different semi-related method seems sort of random. I think it would be better to now just use the trivial definition (`uncompressed_ast(m::Method) = uncompressed_ast(m, m.source)`)